packages/amplify-category-notifications/src/channel-in-app-msg.ts

```diff
@@ -175,22 +172,3 @@ export const pull = async (__context: $TSContext, pinpointApp: $TSAny): Promise<
   pinpointApp[channelName] = channelMeta;
   return buildPinpointChannelResponseSuccess(ChannelAction.PULL, deploymentType, channelName, channelMeta);
 };
-
-/**
- * checks if pinpoint template contains the in app messaging policy
- */
-export const pinpointTemplateHasInAppMessagingPolicy = async (context: $TSContext): Promise<boolean> => {
```

Comment:
moved to analytics category since it is checking a template in analytics category and it is reused in a couple of other places there for the migration purposes